Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "petgraph"
-version = "0.1.5"
+version = "0.1.6"
```

Comment:
I think it's best to save this until I release a new version. I will do directly after this is merged, I promise.